task-definition.json

```diff
@@ -0,0 +1,36 @@
+{
+  "family": "service",
+  "executionRoleArn": "ecsTaskExecutionRole",
+  "containerDefinitions": [
+    {
+      "name": "devops-application",
+      "image": "jasoncarneiro/devops-app:latest",
+      "cpu": 1,
+      "portMappings": [
+        {
+          "hostPort": 3000,
+          "protocol": "tcp",
+          "containerPort": 3000
+        }
+      ],
+      "environment": [
+        {
+          "name": "PORT",
+          "value": "3000"
+        }
+      ],
+      "memory": 768,
```

Comment:
Precisa de tudo pra 1 container?

Reply:
Fiz isso pra forçar pedir mais recurso pro ECS. O ECS não tem um esquema de quando mandar um deploy, ele dropa o container e sobe outro.
O que tive que fazer é diminuir a quantidade minima de containers vivo para 0, e aí o container novo pede bastante recurso (não disponível) fazendo com que drope o que está vivo e suba o novo.

Reply:
Fiz isso pra forçar pedir mais recurso pro ECS. O ECS não tem um esquema de quando mandar um deploy, ele dropa o container e sobe outro.
O que tive que fazer é diminuir a quantidade minima de containers vivo para 0, e aí o container novo pede bastante recurso (não disponível) fazendo com que drope o que está vivo e suba o novo.